### DISCUSSION TITLE
Ignoring files, and files erroneously tracked

### DISCUSSION THREAD

**Discussion by jdaw1** [community/community#3411] [Codespaces] [answered]
New to Git. Using Mac and GitHub desktop. As .DS_Store files kept appearing, I made the schoolboy error of ignoring them. But that doesn’t ignore existing ones: it fails the principle of least surprise. Instead pull requests fail, and cannot be resolved by the web interface — “too complicated” to exclude ignored files. Many google searches later, many commands entered without being understood, and I still cannot update my repo. Sigh.

**Suggestion** (if this is the wrong place for such a suggestion, please instruct where). Please make ignore less hostile. Because there is mystery unhappiness behind the scenes, I do not fully know what this means. But it includes prompts:
* “You have ignored some files already in the repo. Should they be deleted from the repo?”
* “New files ignored will be ignored. Would you also like to ignore existing files?”

If that has technical difficulties, instead:
* “You have ignored some files already in the repo. Do not do this unless you really understand how much trouble this will cause you. Are you sure you want to proceed?”

Thank you.

**Comment by PranamBhat** [comment]
Hi @jdaw1 


[GitHub Discussions](https://docs.github.com/pt/discussions) is currently in beta testing and the Vercel community is invited to participate with us! This is a place to ask questions to the community or request features from our team. You can react to comments just like issues and you can even create threads!

🐞 If you found a bug, it is best to [submit a support request](https://vercel.com/support/request) to ensure our team resolves your issue.

❓ If you have a question, [search the knowledge base](https://vercel.com/support) as well as existing discussions.

🆕 Otherwise, start a new discussion.

As a reminder, the [code of conduct](https://github.com/vercel/vercel/blob/master/.github/CODE_OF_CONDUCT.md) still applies for GitHub Discussions. We want to make this a place that everyone feels welcome!) Discussions is currently in beta testing and the **Vercel** community is invited to participate with us! This is a place to ask questions to the community or request features from our team. You can react to comments just like issues and you can even create threads!


Thank you!

**Comment by 2percentsilk** [comment, marked as answer]
Hey @jdaw1,

I noticed you indicated that you're using GitHub Desktop - is the experience you're encountering within a GitHub Codespace or on your local machine via the GitHub Desktop client?

That will help me route you more appropriately!

**Reply by jdaw1** [reply]
I have been using GitHub desktop. The online recommendations — numerous, heeded — all required use of command line. I think what’s needed is to have desktop forget everything, re-compare to remote server, heed ignore, and re-send updated files. I think.

But I am trying to make bigger point: ‘ignore’ is *de facto* hostile to newbies. It very much fails the principle of least surprise.

PS: [jdaw1/placemat](http://github.com/jdaw1/placemat/).

**Reply by 2percentsilk** [reply]
Thanks for the clarification. 

At this time, this space is currently reserved for feedback on Discussions, GitHub for Mobile, and Codespaces. Given your feedback is about GitHub Desktop, please share your feedback via our [support channel](https://support.github.com/contact/feedback) or our [community forum](https://github.community).

**Reply by jdaw1** [reply]
Deleting everything and reloading seems to have worked. Thank you. 
The support channel wouldn’t let me in, so not submitted. 
Thank you for comments.
